public/src/components/question/question_form.jsx

```diff
@@ -1,10 +1,18 @@
 import React, { Component } from 'react';
-import { Field } from 'redux-form';
+import { Field, FieldArray } from 'redux-form';
 import { Row, Col, Button, ButtonToolbar } from 'react-bootstrap';
 import FieldGroup from '../../components/common/form/field_group';
 import Dropdown from '../../components/common/form/dropdown';
+import QuestionResponses from './question_responses';
 
 export default class QuestionForm extends Component {
+  componentDidMount() {
+    if (!this.props.onSubmit) {
+      throw new Error('onSubmit is not passed. Pass in a function.');
```

Comment:
I wanted this component to throw errors if these two props weren't passed. I researched online and found similar examples, but I don't know if it's the best way. Open to suggestion.